src/modules/config/services/tests/provers-config.service.spec.ts

```
@@ -0,0 +1,127 @@
+  });
+
+  describe('getCcipGasLimit', () => {
+    it('should return configured gas limit', () => {
+      expect(service.getCcipGasLimit()).toBe(300000);
+    });
+
+    it('should throw error when CCIP not configured', () => {
+      mockConfigService.getOrThrow.mockImplementation(() => {
+        throw new Error('Configuration key "provers.ccip" does not exist');
+      });
+      expect(() => service.getCcipGasLimit()).toThrow('Configuration key "provers.ccip" does not exist');
```

ESLint

[error] Replace `'Configuration·key·"provers.ccip"·does·not·exist'` with `⏎········'Configuration·key·"provers.ccip"·does·not·exist',⏎······`

(prettier/prettier)